### PR TITLE
Folio Fixes

### PR DESCRIPTION
**Pull request by paltman**
@gpizzorno diva doesn't seem to respect the event when sending on `Diva.Events.publish` here but noticed in the source code that it was responding to window resize events so I'm faking that.    Would love to get your feedback here before I put more time on fiddling with the viewer.  I'll move on to note rendering in the mean time.  Thanks!

### CODE REVIEW

**Comment by gpizzorno** [comment]
Thanks, @paltman, it seems to work much better overall! A few remaining issues:

- If text in the transcription pane is longer than the height of its container, the entire viewer resizes beyond the window's height. At that point the old issue with the half-cut image re-emerges.

- There appears to be a min-height constraint somewhere. If the window is shrunk past a certain point, the viewer stops resizing accordingly. Like before, when this happens the old cut-off image behaviour re-emerges.

- There's a visual artefact when enlarging the image pane: if the image is zoomed larger than the pane's extents, it appears cut-off until the resizing handle is released (I assume because the resizing event is triggered onDragEnd).

All this is in the vertically split mode (the horizontal one seems unchanged, so I didn't look carefully at it). For reference, I'm using this record to test: https://public.127.0.0.1.xip.io:8443/collections/records/08ef70ff-9195-480e-8b1d-2e5224a7d5da/

**Comment by paltman** [comment]
@gpizzorno addressed some (maybe all of this).  you are right, about horizontal mode.  I think I just need to change around the calcs there but have only been focused thus far on vertical mode.

File: dalme_app/static/css/dalme_public/dalme_public.css [hunk 1]
```diff
@@ -2819,7 +2818,7 @@ section.transcription {
   background-color: #fff;
   flex: 1 1 auto;
   z-index: 1;
-  min-height: calc(100vh - (var(--header-height) + 150px));
+  height: calc(100vh - (var(--header-height) + 88px));
```
Comment (gpizzorno) [inline dalme_app/static/css/dalme_public/dalme_public.css]: Should this be `height: calc(100vh - (var(--header-height) + 96px));`? I'm counting as 54px from the footer + 42px from the toolbar.

`section.transcription` also needs a `min-height: 100px` to match a similar constraint in the default diva-outer class.

File: dalme_app/static/css/dalme_public/dalme_public.css [hunk 2]
```diff
@@ -1267,8 +1266,8 @@ i.inactive {
   color: #4e0f0f;
   background: #f1b5b5;
 }
-#diva_viewer {
-    flex-grow: 1;
+.diva-wrapper {
```
Comment (gpizzorno) [inline dalme_app/static/css/dalme_public/dalme_public.css:1269]: There is another rule for this class (`div.diva-wrapper`) around line 1964. Adding `min-height: calc(100vh - (var(--header-height) + 96px));` (or 88px, see my other comment on this file), to solve issue A in my PR comment.

**Review by gpizzorno — COMMENTED** [review]
I think we're almost there, @paltman! Here are the issues I noticed:

- A: If the height of the window gets shrunk past a certain point (~590px), the footer no longer fits in the window, and at that point the transcription pane continues to shrink as if the footer was still there (i.e. by an additional 54px), leaving a bit of dark background visible. I thought this must be because of some min-height constraint, and it was. There is a diva-wrapper rule imposing a 60vh height limit. Changing that to the same calculation as the height of section.transcription (and adding a separate 100px min-height to match one internal to Diva) fixes the issue. I've annotated the css file accordingly. 

- B: There is a weird Weird visual artefact happening pretty consistently on load: a tile or two of the image fail to load (this seems local, not server-related) as the moment something forces an update the tiles show up without any network activity.

- C: On window resize, the image pane stays the same width, while the transcription pane resizes to take up the extra width. Ideally, they should both resize proportionally.

**Comment by paltman** [comment]
I can't seem to reproduce C:

Screen Shot 2020-12-29 at 10 14 11 AM
Screen Shot 2020-12-29 at 10 14 19 AM
Screen Shot 2020-12-29 at 10 14 30 AM

**Comment by gpizzorno** [comment]
I just tried and I can't reproduce it either, so let's just forget about it. PS: you may want to pull from develop to get an up-to-date version of the code (I've been working on it quite a bit the past week).

**Comment by paltman** [comment]
@gpizzorno just following up to get a since of where we stand.  is the only thing left for me to do is to resolve the conflicts so this is mergeable?

**Comment by gpizzorno** [comment]
@paltman These commits fixed most of the issues with the vertical mode, but there were still a lot of problems with the horizontal view. Plus, there were also a tonne of fixes needed for the TEI rendering (the major being the notes and marginalia). Unfortunately, we're in a rush to go live, so I ended up re-writing a lot of the viewer code from scratch and integrating it into the Vue app. It was the only way to solve all the little issues in both modes, there was just too much superfluous code creating weird interactions. The new code is in the develop branch if you're interested.

**Pull request closed without merging**